src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+:root {
+  --main-color: #00acdc;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use `*` selector as it impacts performance. It's better to set styles only for elements that require them. If you need to reset default margins and paddings, consider using a more specific reset CSS or applying these styles only to the elements that need them.